service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementAboutToSubmitControllerTest.java

```diff
@@ -112,8 +113,8 @@ void shouldAddConfidentialPlacementAndPlacementsWithoutPlacementOrderToCaseDetai
             .isEqualTo(expectedPlacementWithoutConfidentialDocuments(childPlacement, childApplication));
     }
 
-    private List convertToList(Map<String, Object> updatedCaseDetails, String string) {
-        return mapper.convertValue(updatedCaseDetails.get(string), List.class);
+    private List<?> convertToList(Map<String, Object> updatedCaseDetails, String string) {
+        return mapper.convertValue(updatedCaseDetails.get(string), new TypeReference<>() {});
```

Comment:
just curious why we using `TypeReference` why not `List.class`?

Reply:
List just returns a list without any type value, by adding the wildcard to the return type List no longer matches this and as such a type reference is required to convert to the correct type.

Reply:
this is correct. But behind the scenes `mapper.convertValue` uses `TypeReference`

Reply:
will test and see if list works, usually we just use type reference so that is what I did but maybe that is only required when we are requesting a specific type

Reply:
Seems like it works with `List.class` works so will change back to that